lcs/agents/acs2/Classifier.py

```diff
@@ -84,10 +91,22 @@ def __hash__(self):
         return hash((str(self.condition), self.action, str(self.effect)))
 
     def __repr__(self):
-        return "{}-{}-{} @ {}".format(self.condition,
-                                      self.action,
-                                      self.effect,
-                                      hex(id(self)))
+        return "{} {:2} {:16} {:21} q: {:<5.3} r: {:<6.4} ir: {:<6.4} " \
```

Comment:
Please use the f-strings here. It would be more readable.

Reply:
Done :)